vivisect/tools/graphutil.py

```diff
@@ -180,84 +180,14 @@ def getCodePathsThru(fgraph, tgtcbva, loopcnt=0, maxpath=None):
             for node,edge in path:
                 ...etc...
     '''    
-    # this starts with the "To" side, finding a path back from tgtcbva to root
```

Comment:
rework getCodePathsThru() to mimic the much more elegant version of code from visi (in HierGraph.getHierPathsThru)